src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -522,11 +539,8 @@ public IntervalList getLociToGenotype(final Collection<Fingerprint> fingerprints
         // Now go through the data at each locus and figure stuff out!
         for (final SamLocusIterator.LocusInfo info : iterator) {
 
-            // if statement to avoid string building.
-            // TODO: replace with lambda version once htsjdk is rev'ed
-            if (Log.isEnabled(Log.LogLevel.DEBUG)) {
-                log.debug("At locus " + info.toString());
-            }
+            log.debug(() -> "At locus " + info.toString());
```

Comment:
unnecessary `.toString()`
```suggestion
            log.debug(() -> "At locus " + info);
```